### ISSUE TITLE
Migrate scripts to LiteX main repository

### ISSUE THREAD

**Issue by mateusz-holenko** [litex-hub/litex-renode#37]
Since those scripts are very closely related to LiteX we should keep them in the upstream repository to make them:
* easier to verify against changes,
* discoverable by new users.

**Comment by mateusz-holenko** [comment]
We already moved two out of three scripts upstream, the only one left is  https://github.com/litex-hub/litex-renode/blob/master/generate-mocserver-json.py. @CarlFK is it still relevant for you? Do you want it to become a part of LiteX?

**Comment by mateusz-holenko** [comment]
@CarlFK: What is the status of the `generate-mocserver-json.py` script - would you like it to be transferred to the LiteX repo?

**Comment by CarlFK** [comment]
yes, I think it would be good if it lived upstream.  I just reviewed it, looks good to go.

It's purpose is to get a process started that has circular dependencies:  Front end js dev doesn't want to write code where this is no back end, and back end fpga dev doesn't want to write code if there isn't anyone working on the front end.    generate-mocserver-json.py provides the back end so some js can be written. 
 
So the plan is the front end dev gets started and as soon as something looks promising generate-mocserver-json.py will be  implemented in fpga.  I was driving this for the hdmi2usb project, but covid has kinda put that need on hold, thus my involvement is stalled. But the plan was for this to be a generic debugging too for any project, thus it should live here and I'm happy to help with it.

**Comment by mateusz-holenko** [comment]
For the record - created a PR to litex upstream https://github.com/enjoy-digital/litex/pull/1071.
